src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
I think we should to throw a `ParseException` here if the constant does not exist (or at least do that when the `PARSE_EXCEPTION_ON_INVALID_TYPE` flag is set).